.travis.yml

```diff
@@ -4,9 +4,9 @@ python:
   - '3.5'
 
 install:
-  - pip3 install HookTest==0.2.2
+  - pip3 install HookTest==1.0.0
 
-script: hooktest --scheme epidoc --workers 3 --verbose --travis --countword --allowfailure ./ --from_travis_to_hook "http://ci.perseids.org/api/hook/v2.0/user/repositories/OpenGreekAndLatin/csel-dev"
+script: hooktest ./ --scheme epidoc --workers 3 --verbose 10 --manifest --countword --allowfailure --console table --hookUI "http://ci.perseids.org/api/hook/v2.0/user/repositories/OpenGreekAndLatin/csel-dev"
```

Comment:
You should take --allowfailure out of the command line since this is a 100% repo.